crates/crypto_api/src/fake_system/mod.rs

```diff
@@ -17,20 +20,20 @@ impl CryptoRandom for FakeCryptoSystem {
         let mut buffer = buffer.write_lock();
 
         let mut idx = 0;
-        let mut seed = 0;
+        let seed: &mut usize = unsafe { FAKE_TRACK.get_mut() };
```

Comment:
You can also use a `lazy_static!` pattern here which makes the unsafe part not necessary.